Dockerfile

```diff
@@ -3,6 +3,13 @@ FROM ruby:2.6.4
 RUN apt-get update
 RUN apt-get install apt-transport-https
 
+RUN apt-get update && apt-get install sudo && apt-get clean &&\
+    sed -i s+secure_path=.*+secure_path="$PATH"+ /etc/sudoers
+
+RUN curl -fsSL https://deb.nodesource.com/setup_8.x | sudo -E bash - &&\
+    apt-cache policy nodejs &&\
+    apt-get install -y nodejs=8.17.0-1nodesource1
```

Comment:
Manually force version 8 because there is also version 10 in the default apt repository. shouldn't be an issue to pin it because it is EOL for a long time and won't get any update anymore